docs/mkdocs.yml

```
@@ -9,7 +9,7 @@ theme:
   feature:
     tabs: true
 
-strict: true
+strict: false
```

```suggestion
strict: true
```

Lets restore this back to the original